packages/module/patternfly-docs/content/extensions/component-groups/examples/InvalidObject/InvalidObject.md

```diff
@@ -0,0 +1,29 @@
+---
+# Sidenav top-level section
+# should be the same for all markdown files
+section: extensions
+subsection: Component groups
+# Sidenav secondary level section
+# should be the same for all markdown files
+id: InvalidObject
+# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
+source: react
+# If you use typescript, the name of the interface to display props for
+# These are found through the sourceProps function provided in patternfly-docs.source.js
+propComponents: ['InvalidObject']
+---
+
+import InvalidObject from '@patternfly/react-component-groups/dist/dynamic/InvalidObject';
+
+
+This component can be used to display when an error occurs. It shows an error and a button that allows you to go back to the homepage. 
+
+
+
+## Component usage
+
+### Example
+
```

Comment:
```suggestion
## Examples

### Basic invalid object 

A basic invalid object should have a `<Title>` that gives users context on the error and gives them options to navigate the error. These options should include a `<Button>` that redirects users to another page, such as a homepage.

```

This is what I'm gathering from the tsx? lmk if this is wrong

Reply:
I don't think this is right. Title and Button are not options. They're both set in stone as to what they say and do if that makes sense. it just throws an error with a link to the home page.

Reply:
Correct @edonehoo the only option is isBeta prop that will /beta on to the home page when they navigate back by clicking the link/button.

Reply:
@InsaneZein @dlabaj I updated my above suggestion to remove the former phrasing ^^

Reply:
looks good. Thank you!